terraform/deployments/ga4-analytics/service_account.tf

```diff
@@ -0,0 +1,25 @@
+resource "google_service_account" "sa--ga4-analytics-352613" {
+  account_id                   = "ga4-analytics-352613"
```

Comment:
If this is meant to be the same as the project id - could we use a reference to the project id?